src/GitReleaseManager.Core/VcsService.cs

```diff
@@ -177,7 +186,7 @@ private async Task AddAssetsAsync(string owner, string repository, string tagNam
                         if (!release.Body.Contains(_configuration.Create.ShaSectionHeading))
                         {
                             _logger.Debug("Creating SHA section header");
-                            stringBuilder.AppendLine(string.Format("### {0}", _configuration.Create.ShaSectionHeading));
+                            stringBuilder.AppendLine(string.Format(CultureInfo.InvariantCulture, "### {0}", _configuration.Create.ShaSectionHeading));
```

Comment:
```suggestion
                            stringBuilder.AppendFormat(CultureInfo.InvariantCulture, "### {0}", _configuration.Create.ShaSectionHeading).AppendLine();
```

or separate the calls into two calls

```suggestion
                            stringBuilder.AppendFormat(CultureInfo.InvariantCulture, "### {0}", _configuration.Create.ShaSectionHeading);
                            stringBuilder.AppendLine();
```